src/Microsoft.DotNet.Build.Tasks/PackageFiles/toolruntime.targets

```diff
@@ -24,9 +24,12 @@
     <ToolHostCmd Condition="'$(COMPLUS_InstallRoot)' != ''">(set COMPLUS_InstallRoot=) &amp; $(ToolHostCmd)</ToolHostCmd>
   </PropertyGroup>
   
-  <Target Name="EnsureBuildToolsRuntime"
-      Inputs="$(BuildToolsSemaphore)"
-      Outputs="$(ToolRuntimeSempahore)">
+  <Target Name="EnsureBuildToolsRuntime">
+    <Error Condition="!Exists('$(ToolRuntimeSempahore)')"
+           Text="The Tool Runtime directory [$(ToolRuntimePath)] does not exist. Please run $(SourceDir)Build.cmd to ensure the tools are installed before attempting to build an individual project." />
+  </Target>
+
+  <Target Name="RestoreToolRuntime">
 
```

Comment:
Yes, It looks like dotnet-publish works as expected, so I will go ahead and remove this unused target.